Tests/UserServiceTests/Validators/UserEmailPasswordValidatorTests.cs

```diff
@@ -0,0 +1,402 @@
+﻿using System;
+using UserService.Validators;
+using DTO;
+using FluentValidation;
+using NUnit.Framework;
+using System.Linq;
+using DTO.RestRequests;
+
+namespace UserServiceTests.Validators
+{
+    public class UserEmailPasswordValidatorTests
+    {
+        private IValidator<CreateUserRequest> validator = new CreateUserRequestValidator();
+
+        [Test]
+        public void AllFieldsEmpty()
+        {
+            CreateUserRequest user = new CreateUserRequest();
+
+            Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(user));
+        }
+
+        [Test]
+        public void NullEmail()
+        {
+            CreateUserRequest user = new CreateUserRequest
+            {
+                Email = null,
+                Password = "123",
+                FirstName = "Leo",
+                LastName = "Kor",
+                Birthday = DateTime.Today
+            };
+
+            var validationResult = Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(user));
+
+            var expectedError =
+                validationResult.Errors.FirstOrDefault(
+                    error => error.ErrorMessage == ErrorsMessages.EmailEmpty);
+
+            Assert.IsNotNull(expectedError);
+        }
+
+        [Test]
+        public void EmptyEmail()
+        {
+            CreateUserRequest user = new CreateUserRequest {
+                Email = "",
+                Password = "123",
+                FirstName = "Leo",
+                LastName = "Kor",
+                Birthday = DateTime.Today
+            };
+
+            var validationResult = Assert.Throws<ValidationException>(() => validator.ValidateAndThrow(user));
+
+            var expectedError =
+                validationResult.Errors.FirstOrDefault(
+                    error => error.ErrorMessage == ErrorsMessages.EmailEmpty);
+
+            Assert.IsNotNull(expectedError);
+        }
+
+        [Test]
+        public void WrongEmail()
```

Comment:
Mb `InvalidEmail` would be a better name, but I'm not sure

Reply:
fixed